### DISCUSSION TITLE
Can I skip from switching to base branch?

### DISCUSSION THREAD

**Discussion by gadingnst** [ArtiomTr/jest-coverage-report-action#247] [Q&A] [answered]
Hello, I want to skip from switching branch, only running coverage on the current branch, how can I do that?

**Comment by ArtiomTr** [comment, marked as answer]
Hello @sutanlab 👋,

Collecting coverage **only** for the head branch is not "normal" behavior of this action. However, you can achieve this - [it is described here](https://github.com/ArtiomTr/jest-coverage-report-action#opt-out-coverage-comparison-features).
